MAINTAINERS

```diff
@@ -138,10 +138,10 @@ W: http://plugins.geany.org/geanyprj.html
 S: Odd Fixes
 
 geanypy
-P: Lex Trotman <elextr@gmail.com>
-M: Lex Trotman <elextr@gmail.com>
+P:
+M:
 W: http://plugins.geany.org/geanypy.html
-S: Maintained
```

Comment:
The "diverged from upstream" part can be easily fixed by a simple "git revert" of the patch I merged or by applying the patch upstream - but this requires some kind of actual maintenance of the plugin which is something that should happen if the plugin should stay in GP (my opinion).